openapi/p4pa-pu-sil.openapi.yaml

```diff
@@ -182,6 +216,19 @@ components:
           type: string
           format: date-time
           description: ISO 8601 date of completion due to expiry of deadline
+    NotifyPaymentDTO:
+      type: object
+      properties:
+        outcome:
+          type: string
+          enum:
+            - OK
+            - KO
+        faultCode:
+          type: string
+        faultDescription:
+          type:
+            string
```

Comment:
let's avoid to return error data with the normal body... let's use the ErrorDTO instead... if we have not relevant information together with the 200 status, let's not configure return body

Reply:
This model is not used, let's remove it!
For internal API we will use PuSilErrorDTO (let's relate it to API's error code)

For external API, probably we should introduce a new ErrorDTO having dedicated error code... but we could see it forward